qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -230,7 +235,7 @@ def cancel(self) -> bool:
             might not be possible depending on the environment.
 
         Raises:
-            JobError: if the job has not been submitted or if there was
+            IBMQJobApiError: if the job has not been submitted or if there was
```

Comment:
I know it's there before, but `IBMQJobApiError` is no longer raised for "job has not been submitted", only for server errors.